addons/web/static/src/webclient/navbar/navbar.xml

```diff
@@ -50,7 +50,8 @@
         t-foreach="apps"
         t-as="app"
         t-key="app.id"
-        class="{ 'o_app': true, focus: menuService.getCurrentApp() === app }"
+        t-if="!app.technical_chm || app.technical_chm == this.technical_chm.active"
```

Comment:
t-if="!app.technical_chm || this.technical_chm.active"

Reply:
No, because we can hide some feature in technical mode (technical-chm="0").
It's used in accounting feature (like !group_no_one or with strange fields)